mllib/src/test/scala/org/apache/spark/mllib/rdd/MLPairRDDFunctionsSuite.scala

```diff
@@ -33,4 +33,16 @@ class MLPairRDDFunctionsSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(topMap(3) === Array(7, 5, 2))
     assert(topMap(5) === Array(1))
   }
+
+  test("topByKeyUsingBuckets") {
```

Comment:
BTW, does this test the number of reducers? It looks the input/output is the same with the test above.